sycl/source/detail/device_info.hpp

```diff
@@ -588,6 +588,7 @@ struct get_device_info_impl<range<Dimensions>,
   X("gfx906", oneapi_exp_arch::amd_gpu_gfx906)                                 \
   X("gfx908", oneapi_exp_arch::amd_gpu_gfx908)                                 \
   X("gfx90a", oneapi_exp_arch::amd_gpu_gfx90a)                                 \
+  X("gfx90c", oneapi_exp_arch::amd_gpu_gfx90c)                                 \
```

Comment:
@jinz2014 compilation failed, `amd_gpu_gfx90c` should be added also to `enum class architecture` in sycl/include/sycl/ext/oneapi/experimental/device_architecture.hpp

Reply:
Thanks.